templates/setup.yml

```diff
@@ -411,6 +411,10 @@
     - role: galaxy/auxiliary
     # /role-specific:auxiliary
 
+    # role-specific:matterbridge
+    - role: galaxy/matterbridge
```

Comment:
Some comments on the role itself:

- in `templates/matterbridge.service.j2`, the `--mount type=bind,src={{ matterbridge_toml_path }},dst=/etc/matterbridge/matterbridge.toml \` line could probably benefit from `,readonly`. If the bridge needs the file to be writable, perhaps it can be documented there.

- if the Matterbridge container can start as `--read-only`, it'd be good to run it that way. If not, please document in `matterbridge.service` why it's not done. Sometimes mounting a tmpfs somewhere may let a service start in a readonly container ([example from Plausible](https://github.com/mother-of-all-self-hosting/ansible-role-plausible/blob/d245e96d0f88e22892daf34d495ecc28b5ee5206/templates/systemd/plausible.service.j2#L35)), while other times it's not possible.

- there are no systemd requires/wants definitions for `matterbridge.service`. We tend to make all our roles depend on `docker.service`. Recently, we started using `devture_systemd_docker_base_docker_service_name` instead of hardcoding `docker.service`. Refer to the [Plausible role](https://github.com/mother-of-all-self-hosting/ansible-role-plausible) for an example ([here](https://github.com/mother-of-all-self-hosting/ansible-role-plausible/blob/d245e96d0f88e22892daf34d495ecc28b5ee5206/defaults/main.yml#L194-L204) and [here](https://github.com/mother-of-all-self-hosting/ansible-role-plausible/blob/d245e96d0f88e22892daf34d495ecc28b5ee5206/templates/systemd/plausible.service.j2#L4-L11)).

- the Matterbridge link in `README.md` is currently broken

Reply:
I have incorporated your feedback and believe I have implemented it as you intended. Would you mind taking another look at it?

Reply:
It looks good now!

The only thing I've noticed is that `matterbridge_config_dir_path` is actually pointed to a directory named `data` which is confusing.

Many of the roles have an Ansible-managed config (dumped into a `config` directory), as well as some `data` directory which contains container-generated data. Naming the config directory `data` is a source of confusion.

It'd be great to change this now, before people start using it (and having to worry about a migration path).

Reply:
I agree that this is confusing. I changed this in the role repository :+1: